app/models.py

```diff
@@ -1198,6 +1217,14 @@ def get_link(self):
             _external=True,
         )
 
+    @property
+    def template_process_type(self):
+        if self.template_type == SMS_TYPE and self.template_categories.sms_process_type:
```

Comment:
I think this needs to be self.category.sms_process_type IF self.process_type is empty? 
Can we add some tests to this

Reply:
Yea you're right, I had the logic correct the first time and then changed it 🤦